recipes/g2o-python/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

Reply:
This file is still here. Why resolve this comment?